ppb_vector/vector2.py

```diff
@@ -487,7 +487,7 @@ def truncate(self: Vector, max_length: typing.SupportsFloat) -> Vector:
 
         Note that :py:meth:`vector.scale(max_length) <scale>` is equivalent to
         :py:meth:`vector.truncate(max_length) <truncate>` when
-        :py:meth:`max_length \< vector.length <length>`.
+        :py:meth:`max_length < vector.length <length>`.
```

Comment:
This doesn't render right in Sphinx.

Reply:
Ohno.  I thought I would dodge the issue by using a unicode `<` sign, rather than the angled braket  ;_;
Fixed with a different symbol  O:3